soem/ethercattype.h

```diff
@@ -513,10 +513,10 @@ typedef struct
 #define put_unaligned64(val, ptr)        \
   (memcpy((ptr), &(val), 8))
 
-#if !defined(EC_BIG_ENDIAN) && defined(EC_LITTLE_ENDIAN)
-  #define put_unaligned32(val, ptr)        \
-    (memcpy((ptr), &(val), 4))
+#define put_unaligned32(val, ptr)        \
```

Comment:
Can you revert the changes in this file all together? Doesn't need to be part of the PR anymore.?

Reply:
Done